### ISSUE TITLE
boxed values arrays can't be transfered to a dataset

### ISSUE THREAD

**Issue by genmeblog** [techascent/tech.ml.dataset#16]
```clojure
(let [arr (into-array Double (repeat 10 1.0))]
  (tech.ml.dataset/name-values-seq->dataset  [["double" arr]]))

(let [arr (into-array Boolean (repeat 10 true))]
  (tech.ml.dataset/name-values-seq->dataset  [["boolean" arr]]))
```

**Comment by genmeblog** [comment]
also

```clojure
(let [arr [true nil false]]
  (tech.ml.dataset/name-values-seq->dataset  [["boolean" arr]]))
```

**Comment by genmeblog** [comment]
I need to have a possibility to mark missing values in boolean column (Tablesaw supports this)

**Comment by cnuernber** [comment]
Tentatively fixed in any version 2.0+
